packages/components/src/tabs/styles.ts

```diff
@@ -76,10 +76,11 @@ export const Tab = styled( Ariakit.Tab )`
 		font-weight: 500;
 		text-align: inherit;
 		hyphens: auto;
+		color: ${ COLORS.theme.foreground };
 
 		&[aria-disabled='true'] {
 			cursor: default;
-			opacity: 0.3;
+			color: ${ COLORS.theme.gray[ 700 ] };
```

Comment:
Is it intentional that we're making the text so close to the non-disabled tab text color now? I'd likely prefer a lighter gray, but as you said, let's ask for some design feedback.

Reply:
Yup! Looking at https://github.com/WordPress/gutenberg/pull/62480#discussion_r1684493572, there seems to be consensus on the lighter `600` shade. I will update accordingly.

Reply:
Updated, using a semantic variable that aliases to gray 600.